In [1]:
import numpy as nm  
import matplotlib.pyplot as mtp  
import pandas as pd  

### NUMBER OF VITAL EVENTS BY SEX, REGION, PROVINCE, AND CITY/MUNICIPALITY, PHILIPPINES: JANUARY TO SEPTEMBER 2020 (Preliminary)													
### As of 30 December 2020													


### RAW DATA

In [2]:
data= pd.read_csv('VStable.csv')  
data

REGION, PROVINCE, CITY/MUNICIPALITY  \
0                                           PHILIPPINES   
1                         NATIONAL CAPITAL REGION (NCR)   
2                                        FIRST DISTRICT   
3                                        City of Manila   
4                                       SECOND DISTRICT   
...                                                 ...   
1738  Source: Philippine Statistics Authority, Civil...   
1739  Note: Figures are results of actual registrati...   
1740  Civil Registration is a continuous process. He...   
1741  * Includes babies born in the Philippines whos...   
1742  ** Includes persons died in the Philippines wh...   

     Total Number of Live Births by Sex and by Usual Residence of Mother*   \
0                                             1,038,711                      
1                                               137,616                      
2                                                19,702                      
3                                                19,702                      
4                                                49,007                      
...                                                 ...                      
1738                                                NaN                      
1739                                                NaN                      
1740                                                NaN                      
1741                                                NaN                      
1742                                                NaN                      

         Male   Female  \
0     542,285  496,426   
1      71,671   65,945   
2      10,341    9,361   
3      10,341    9,361   
4      25,561   23,446   
...       ...      ...   
1738      NaN      NaN   
1739      NaN      NaN   
1740      NaN      NaN   
1741      NaN      NaN   
1742      NaN      NaN   

      Total Number of Deaths by Sex and by Usual Residence of Deceased**   \
0                                               448,398                     
1                                                64,669                     
2                                                10,177                     
3                                                10,177                     
4                                                22,558                     
...                                                 ...                     
1738                                                NaN                     
1739                                                NaN                     
1740                                                NaN                     
1741                                                NaN                     
1742                                                NaN                     

        Male1  Female2 Total Number of Marriages by Place of Occurrence  
0     255,960  192,438                                          153,361  
1      36,849   27,820                                           19,526  
2       5,825    4,352                                            3,561  
3       5,825    4,352                                            3,561  
4      12,686    9,872                                            8,354  
...       ...      ...                                              ...  
1738      NaN      NaN                                              NaN  
1739      NaN      NaN                                              NaN  
1740      NaN      NaN                                              NaN  
1741      NaN      NaN                                              NaN  
1742      NaN      NaN                                              NaN  

[1743 rows x 8 columns]

### THE CODE BELOW WILL OUTPUT ONLY THE REPRESENTING REGIONS AND REMOVES INFORMATION BELOW ABOUT THE DATA SET WHICH ARE INDEX 1739 AND ABOVE

In [3]:
data[0:1738]

REGION, PROVINCE, CITY/MUNICIPALITY  \
0                            PHILIPPINES   
1          NATIONAL CAPITAL REGION (NCR)   
2                         FIRST DISTRICT   
3                         City of Manila   
4                        SECOND DISTRICT   
...                                  ...   
1733                           Sitangkai   
1734                         South Ubian   
1735                            Tandubas   
1736                      Turtle Islands   
1737                     FOREIGN COUNTRY   

     Total Number of Live Births by Sex and by Usual Residence of Mother*   \
0                                             1,038,711                      
1                                               137,616                      
2                                                19,702                      
3                                                19,702                      
4                                                49,007                      
...                                                 ...                      
1733                                                 45                      
1734                                                 31                      
1735                                                 77                      
1736                                               -                         
1737                                                 72                      

         Male   Female  \
0     542,285  496,426   
1      71,671   65,945   
2      10,341    9,361   
3      10,341    9,361   
4      25,561   23,446   
...       ...      ...   
1733       18       27   
1734       17       14   
1735       39       38   
1736     -        -      
1737       36       36   

      Total Number of Deaths by Sex and by Usual Residence of Deceased**   \
0                                               448,398                     
1                                                64,669                     
2                                                10,177                     
3                                                10,177                     
4                                                22,558                     
...                                                 ...                     
1733                                                 27                     
1734                                                  2                     
1735                                                 12                     
1736                                               -                        
1737                                                303                     

        Male1  Female2 Total Number of Marriages by Place of Occurrence  
0     255,960  192,438                                          153,361  
1      36,849   27,820                                           19,526  
2       5,825    4,352                                            3,561  
3       5,825    4,352                                            3,561  
4      12,686    9,872                                            8,354  
...       ...      ...                                              ...  
1733       14       13                                                3  
1734        1        1                                                2  
1735        6        6                                                4  
1736     -        -                                                -     
1737      234       69                                             -     

[1738 rows x 8 columns]

### THE CODE BELOW WILL OUTPUT VITAL EVENTS SPECIFIC TO THE VISAYAN REGION

### IT WILL ALSO REMOVE ROW THAT HAD DATA COMBINED OR COMPOSITE SUCH AS THE REGION NAMES AND PHILIPPINES ALONG WITH THE CITIES EXCEPT FOR CITIES THAT DOES NOT HAVE MUNICIPALITIES

In [18]:
visayas_region= data[822:1249].drop(data.index[[822,823,843,862,880,886,930,964,965,1014,1065,1091,1101,
                                                1102,1111,1135,1178,1203,1230]])
visayas_region

REGION, PROVINCE, CITY/MUNICIPALITY  \
824                              Altavas   
825                               Balete   
826                                Banga   
827                                Batan   
828                             Buruanga   
...                                  ...   
1244                         San Ricardo   
1245                              Silago   
1246                               Sogod   
1247                         Tomas Oppus   
1248                    CITY OF TACLOBAN   

     Total Number of Live Births by Sex and by Usual Residence of Mother*   \
824                                                 214                      
825                                                 314                      
826                                                 411                      
827                                                 243                      
828                                                 133                      
...                                                 ...                      
1244                                                 86                      
1245                                                103                      
1246                                                452                      
1247                                                147                      
1248                                              2,558                      

       Male Female  \
824     123     91   
825     165    149   
826     212    199   
827     117    126   
828      74     59   
...     ...    ...   
1244     44     42   
1245     49     54   
1246    235    217   
1247     84     63   
1248  1,362  1,196   

      Total Number of Deaths by Sex and by Usual Residence of Deceased**   \
824                                                 124                     
825                                                 122                     
826                                                 237                     
827                                                 246                     
828                                                  80                     
...                                                 ...                     
1244                                                 34                     
1245                                                 57                     
1246                                                207                     
1247                                                 86                     
1248                                                880                     

     Male1 Female2 Total Number of Marriages by Place of Occurrence  
824     69      55                                               39  
825     74      48                                               30  
826    131     106                                               64  
827    138     108                                               28  
828     47      33                                               19  
...    ...     ...                                              ...  
1244    19      15                                               46  
1245    26      31                                               12  
1246   118      89                                               45  
1247    50      36                                               20  
1248   497     383                                              270  

[408 rows x 8 columns]